cluster/gce/gci/configure-helper.sh

```diff
@@ -520,10 +520,21 @@ function start-etcd-servers {
     rm -f /etc/init.d/etcd
   fi
   prepare-log-file /var/log/etcd.log
-  prepare-etcd-manifest "" "4001" "2380" "200m" "etcd.manifest"
+  prepare-etcd-manifest "" "4001" "2380" "150m" "etcd.manifest"
 
   prepare-log-file /var/log/etcd-events.log
-  prepare-etcd-manifest "-events" "4002" "2381" "100m" "etcd-events.manifest"
+  prepare-etcd-manifest "-events" "4002" "2381" "50m" "etcd-events.manifest"
+}
+
+# Starts Calico policy controller pod.
+# Specifically, replaces the variable values in the manifest, and copies it to
+# /etc/kubernetes/manifests.
+function start-calico-policy-controller {
+  local -r temp_file="/tmp/calico-policy-controller.manifest"
+  cp "${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty/calico-policy-controller.manifest" "${temp_file}"
+  PUBLIC_IP=$(ifconfig eth0 | grep 'inet ' | awk '{print $2}')
+  sed -i -e "s@{{ *grains\.id *}}@$PUBLIC_IP@g" "${temp_file}"
```

Comment:
I used the jinja syntax because the manifests live in the salt part of cluster/.  The GCI scripts use sed to process these files (see prepare-etcd-manifest for an example).

I could put the manifest somewhere else and use a different syntax, but this way felt most consistent: all the manifests are under cluster/saltbase/salt and are handled this way; so I was reluctant to create a new location.  It is definitely a bit strange though!